core/logger/doc.go

```diff
@@ -0,0 +1,7 @@
+// package logger exports multiple type loggers:
+// - the *Logger type is a wrapper over uber/zap#SugaredLogger with added conditional utilities.
```

Comment:
Why not use this one everywhere?

Reply:
It is used everywhere. Default is a *Logger instance and ProductionLogger, CreateTestLogger and CreateMemoryTestLogger are all producing instances of *Logger.

Maybe I should add more about how to use it instead of what it is!?